kroxylicious-additional-filters/kroxylicious-schema-validation/pom.xml

```diff
@@ -15,7 +15,7 @@
         <groupId>io.kroxylicious</groupId>
         <artifactId>kroxylicious-parent</artifactId>
         <version>0.2.0-SNAPSHOT</version>
-        <relativePath>../pom.xml</relativePath>
+        <relativePath>../../pom.xml</relativePath>
```

Comment:
Should this point at the new additional-filters module as parent now?

Reply:
Good catch. That was fine before I added the enforcer rules.